metasyn/distribution/discrete.py

```diff
@@ -146,8 +146,8 @@ def _fit(cls, values):
         lower = values.min()
         high = values.max() + 1
         if len(values) == high-lower and np.all(values.to_numpy() == np.arange(lower, high)):
```

Comment:
Maybe this comparison is slightly inefficient?

Reply:
How would you prefer to do it?

Reply:
not sure, in R I'd do `all(diff(values) == 1)` then we don't need to instantiate a completely new vector which might be a million values. But it's a super small comment which you could ignore as well

Reply:
I don't actually know if `np.arange` will instantiate a new vector. Let's for now keep it like it is. I don't think this will really ever be a bottleneck, and if so we can revise then.